src/Compilers/CSharp/Test/Symbol/DocumentationComments/CrefTests.cs

```diff
@@ -3357,17 +3357,17 @@ private class Inner { }
                 "T",
                 "void C<T>.M()",
                 "C<T>",
-                "Outer",
 
                 // Boring
                 "System",
-                "Microsoft",
 
                 // Inaccessible and boring
                 "FXAssembly",
                 "ThisAssembly",
                 "AssemblyRef",
-                "SRETW");
+                "SRETW",
+                "Outer",
+                "Microsoft");
```

Comment:
This appears to just be a result of the diff between the ref and impl assemblies we were using. Wanted someone else to look at this to verify though.